### ISSUE TITLE
Open Windows Terminal with right-click context menu can't load environment variables

### ISSUE THREAD

**Issue by tzzs** [microsoft/terminal#11378]
### Windows Terminal version (or Windows build number)

10.0.22000.194, 1.10.2383.0

### Steps to reproduce

1. Open Windows Terminal with right-click context menu
2. It will be return the following message when I input npm
```shell
npm: The term 'npm' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
3. Which could load environment variables normally when I open Windows Terminal with right-click start menu.

### Expected Behavior

1. right-click context menu
![image](https://user-images.githubusercontent.com/24679653/135404637-a91e7c76-3bc3-4d65-9ceb-9eaa3503aa3d.png)

2. start menu
![image](https://user-images.githubusercontent.com/24679653/135404575-7c9dd486-cf2b-4b68-81f5-00fa169c57ec.png)

### Actual Behavior

Can't find npm env variable.

**Comment by heysujal** [comment]
I think you don't have node installed or didn't add it to environment variables.
https://stackoverflow.com/questions/20992723/npm-is-not-recognized-as-internal-or-external-command-operable-program-or-bat/47524025
@tzzs  this will help you out.

**Comment by zadjii-msft** [comment]
It might be possible that launching the Terminal from explorer like that doesn't reload updated environment variables. In the Terminal that's launched from explorer, can you do a `echo %path%` (or `echo $env:Path` in powershell)?

**Comment by batero2000** [comment]
Exactly, I have the same bug and a Microsoft worker just said to me to repost here...
![image](https://user-images.githubusercontent.com/14956589/141232672-e9772eb4-3037-4acb-b222-f86906867c69.png)

**Comment by zadjii-msft** [comment]
omg, I am genuinely surprised that actually worked, that's great to hear! 

But also, concerned that this isn't working for you, even after a reboot. 
@batero2000:
* What's your OS version?
* your Terminal version?
* what's `echo %PATH%` inside of Terminal when launched from the explorer context menu?
* `echo %PATH%` when launched normally (like, from the Start Menu)?

**Comment by batero2000** [comment]
> * What's your OS version?
Windows 11 Pro ver. 21H2 (comp. 22000.348)
> * your Terminal version?
Windows Terminal ver. 1.11.2921.0
> * what's `echo %PATH%` inside of Terminal when launched from the explorer context menu?
I just get the error (in spanish quote) `[error 0x80070002 al inicial 'cmd.exe']`
> * `echo %PATH%` when launched normally (like, from the Start Menu)?
`C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files\dotnet\;C:\Users\Federico\AppData\Local\Microsoft\WindowsApps;C:\Program Files\ADB;C:\WINDOWS\System32\WindowsPowerShell\v1.0;"C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\system32\WBEM"`

**Comment by zadjii-msft** [comment]
@batero2000 Your PATH looks kinda messed up... There's no system32 in there at all!

You might want to add that back to your PATH. You can use this dialog:
![image](https://user-images.githubusercontent.com/18356694/145088508-e63549e0-659a-4fe8-bd3b-cb72dcba97c4.png)

(you may need to add some other paths as well)

**Comment by batero2000** [comment]
> @batero2000 Your PATH looks kinda messed up... There's no system32 in there at all!
> 
> You might want to add that back to your PATH. You can use this dialog: ![image](https://user-images.githubusercontent.com/18356694/145088508-e63549e0-659a-4fe8-bd3b-cb72dcba97c4.png)
> 
> (you may need to add some other paths as well)

It worked! I thought that that %systemroot%\system32 was doing the job, turns out wasn't the case.

Thank you pretty much indeed.
